getting_started.ipynb

```diff
@@ -44,7 +36,7 @@
    "id": "49d7b471-cf37-4f74-8d50-b29fd6f5bbc1",
    "metadata": {},
    "source": [
-    "To use *climakitae* (the python 'climate kit' library containing our AE tools), load a new application:"
+    "To use climakitae (the python 'climate kit' library containing our AE tools), load a new application:"
```

Comment:
I think the description in parentheses here is a bit redundant. Would recommend deleting.